### DISCUSSION TITLE
M5stack CoreS3 support?

### DISCUSSION THREAD

**Discussion by AlanB74** [meshtastic/firmware#4873] [Ideas]
Do we think there is a chance to get a port to the M5stack CoreS3 with a M5stack LoRa v1.1 module? I've tinkerered but my coding skills and understanding are still way too novice to achieve anything 😂 dying to utilise the LoRa module before it becomes a paperweight 😭

**Comment by Coopersmith-24601** [comment]
This is already being worked on by multiple people.
